src/MOI_wrapper.jl

```diff
@@ -489,7 +510,8 @@ struct ConstraintConflictStatus <: MOI.AbstractConstraintAttribute end
 MOI.is_set_by_optimize(::ConstraintConflictStatus) = true
 
 function MOI.get(model::Optimizer, ::ConstraintConflictStatus, index::MOI.ConstraintIndex) 
-    return index in model.conflict.rowind
+    _ensure_conflict_computed(model)
+    return index.value in model.conflict.rowind
```

Comment:
`index.value` isn't guaranteed to be the column. You need:
```julia
LQOI.get_column(model, index) in model.confict.rowind
```

Reply:
Actually, that's the issue I was working on, thanks!

Reply:
For constraints, however, it does not work as well as expected: I could not find a `get_row` or similar, but rather a `ConstraintMapping` structure (accessible through `cmap`), which is too complicated to work with (there should be a helper method in LQOI to dispatch on the type of constraint). And `index.value` does not seem to be working well enough...